src/index.html

```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>To-do List</title>
```

Comment:
- [ ] to keep your repo clean , kindly remove any commented code from your project 🙏

Reply:
Hey, thanks for the feedback. I've made the required changes. I would like to know how you are able to add the gif to the review markdown. Anytime I try it, it doesn't show up

Reply:
you can upload it normally as a file 🙏